projects/dnd/src/services/dnd.service.ts

```diff
@@ -96,18 +95,15 @@ export class DndService implements OnDestroy {
     this.sourceHost.clearSelectedDndItems();
 
     const targetDndItemElement: DndItemHtmlElement | null = getDndTargetItemFromEvent(event.srcEvent);
-    if (isNil(targetDndItemElement)) {
-      return;
-    }
 
     const targetHostElement: DndHostHtmlElement = getDndHostElementFromEvent(event.srcEvent);
```

Comment:
Сразу не заметил, т.к. судя по коду ниже элемент может быть null, стоит поправить тип
```suggestion
    const targetHostElement: DndHostHtmlElement | null = getDndHostElementFromEvent(event.srcEvent);
```